train_on_strand.sh

```diff
@@ -16,9 +16,11 @@
 #SBATCH --exclusive
 #SBATCH --output=slurm_output/slurm-%j.out
 
-# 300 seconds before training ends resubmit the job
+## 300 seconds before training ends resubmit the job
 #SBATCH --signal=SIGUSR1@300
 
+## create SLURM output directory if it doesn't exist
+mkdir -p slurm_output
```

Comment:
Well that solution seems way cleaner than my proposed one. Nice!

Reply:
I tried this and it doesn't work...

Reply:
changed to a comment